src/util/Log.h

```diff
@@ -123,28 +121,24 @@ class Log {
     return std::move(ss).str();
   }
 
-  template <unsigned char LEVEL>
-  static string getLevel() {
-    if (LEVEL == TRACE) {
+  template <LogLevel LEVEL>
```

Comment:
While you are at it:
I think there should be a simpler way to format the date above than to explicitly deal with the milliseconds. Do you want to do this as part of your cleanup PR as well?

Reply:
I checked,  but there isn't any syntax to represent milliseconds in the format string